addons/main/functions/fnc_addJammer.sqf

```diff
@@ -44,8 +47,13 @@ if (typeof _unit == "Land_DataTerminal_01_F") then {
 	}];
 };
 
-// add to map, netId is key		jammer, radius, strength, and enabled
-GVAR(jamMap) set [_netId, [_unit, _rad, _strength, true]];
+// add to map, netId is key		jammer, radius, strength, enabled and capabilities
+GVAR(jamMap) set [_netId, [_unit, _rad, _strength, _enabled, _capabilities]];
+
+if (_enabled) then {
+	// add jammer as a signal beacon (so that it can be tracked down with the Spectrum Device)
+	[QEGVAR(spectrum,addBeacon), [_unit, 433, 300, "sweep"]] call CBA_fnc_globalEventJIP;
```

Comment:
I think this can be part of the current issue. This ``addJammer`` function is called with a ``globalEventJIP``, which means every single client will execute what it have. This in turns means that if its enabled, the ``addBeacon`` gets executed for every single client connected to the game multiplied by the amount of players. 
So with 4 players online, it would be called ~16 times for the same jammer. 

It should work for the ``GVAR(jamMap)`` because that is a local variable that gets updated.

I need to have a closer look, but might be possible to add ``isServer`` to the if-statement, so signal is only added if enabled and from the server-instance. Then the addBeacon ``globalEventJIP`` gets sent to all other clients. 
Alternatively might be possible to just run the ``addBeacon`` function locally for everyone, and skip the event broadcast.

Reply:
The CBA guys recommend the following [here](https://github.com/CBATeam/CBA_A3/wiki/Custom-Events-System):
_For state-changes, which you would want synced on JIP, use the standard `publicVariable` and `addPublicVariableEventHandler` commands instead._

This sounds like it applies very well to our situation with the jammer state. I don't see a reason why we would need the [stacking feature](https://cbateam.github.io/CBA_A3/docs/files/events/fnc_globalEventJIP-sqf.html) of `globalEventJIP`. Noone needs the information how many times a jammer has been turned on and off before JIPing. They only need to know the current state.